### PR TITLE
DAOS-15337 test: updated timeout from 60 to 90

### PR DESCRIPTION
**Pull request by arorasamarth**
Test-tag: TestWithTelemetryBasic
Skip-unit-tests: true
Skip-fault-injection-test: true
Priority: 2

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by rpadma2 — APPROVED** [review]

File: src/tests/ftest/control/dmg_telemetry_basic.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ hosts:
   test_servers: 2
   test_clients: 1
 timeouts:
-  test_telemetry_list: 60
+  test_telemetry_list: 90
```
Comment (mjean308) [inline src/tests/ftest/control/dmg_telemetry_basic.yaml:5]: The test was killed after 120 sec (90sec test timeout + 30sec that is added for agent/server start/stop to every test).  It looks like the test was still running at 120 sec.    Try a debug run were you increase the timeout to 600 sec and then determine the amount of time the test actually takes to complete and see if this completion time is reasonable.

If the test used to pass in 60 secs and now it is taking 300 sec then maybe there is another issue that should be looked into.

Comment (daltonbohning) [reply src/tests/ftest/control/dmg_telemetry_basic.yaml:5]: @mjean308 See my comment here
https://daosio.atlassian.net/browse/DAOS-15337

The problem is related to general VM slowness - not this test in particular

**Review by daltonbohning — CHANGES_REQUESTED** [review]
Per https://daosio.atlassian.net/browse/DAOS-15337, increasing the timeout isn't the solution.

@phender Has #13925 to add more debugging for the general issue. And increases the timeout for ALL tests as a workaround in the meantime

**Comment by daltonbohning** [comment]
Closing since ticket is resolved: https://daosio.atlassian.net/browse/DAOS-15337

**Pull request closed without merging**